app/controllers/service_providers_controller.rb

```diff
@@ -114,6 +114,7 @@ def service_provider_params
       :sp_initiated_login_url,
       :group_id,
       :identity_protocol,
+      :production_issuer,
```

Comment:
Looks like there's a mass assign issue here. A non-admin user can add a value for `production_issuer` to the request and set the production issuer.

Reply:
If we are going to allow them to modify it, then let's not hide the field. If we aren't going to allow them to modify if, then let's follow through in the controller and build some tests around it.

Reply:
I opted to not let them modify it.  It would be too confusing for them and they might get the information wrong.